pkg/sql/schemachanger/scop/ops.go

```diff
@@ -30,6 +30,13 @@ const (
 	// ValidationType represents constraint and unique index validations
 	// performed using internal queries.
 	ValidationType
+
+	maxType
```

Comment:
This isn't a max, though, it's the max + 1...

In these cases where the enum is unlikely to change in the future it's better to have `const maxType = ValidationType` instead. The alternative (for enums which often have values added to them) is to define a `$<whateverEnumType>ValuesCount` along with `const max$<whateverEnumType> = $<whateverEnumType>ValuesCount - 1`.

Reply:
meh I fixed the max definition